schema-loader/src/test/java/com/scalar/db/schemaloader/SchemaLoaderTest.java

```diff
@@ -0,0 +1,196 @@
+package com.scalar.db.schemaloader;
+
+import static org.mockito.Mockito.verify;
+
+import com.scalar.db.schemaloader.core.SchemaOperator;
+import com.scalar.db.schemaloader.core.SchemaOperatorFactory;
+import java.nio.file.Path;
+import java.util.Collections;
+import java.util.Properties;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mock;
+import org.mockito.MockedStatic;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+
+public class SchemaLoaderTest {
```

Comment:
Can you please add tests for all variations of the `SchemaLoader.load()`/`SchemaLoader.unload()` methods? Also, please add tests with `null` value for the schema file parameter.

Reply:
I added them in [commit](https://github.com/scalar-labs/scalardb/pull/366/commits/0c8f3fe47b504108058a0f419a42bb8a6808f80d)